Core/Core/UITestHelpers/MockDistantURLSession.swift

```diff
@@ -96,8 +96,15 @@ public class MockDistantURLSession: URLSession {
         let task = MockDataTask()
         task.mock = MockDistantURLSession.dataMocks[request.url!]
         if task.mock == nil {
-            print("⚠️ mock not found for url: \(request.url?.absoluteString ?? "<n/a>")")
+            let urlStr = request.url?.absoluteString ?? "<n/a>"
+            print("⚠️ mock not found for url: \(urlStr)")
             print(MockDistantURLSession.dataMocks.keys)
+
+            task.mock = MockData(
+                    data: nil,
+                    response: request.url.flatMap { HTTPURLResponse(url: $0, statusCode: 401, httpVersion: "HTTP/1.1", headerFields: [:]) },
+                    error: NSError.instructureError("no mock found for \(urlStr)"),
+                    noCallback: false)
```

Comment:
this looks like a good idea, maybe a `501` error. This will conflict with some other changes for the moment though, so can we remove this?